src/terra-dev-site/guide/CreatingItemCollection.f.guide.mdx

```diff
@@ -0,0 +1,187 @@
+import ResponsiveListandTableExample from '../example/ResponsiveListandTableExample?dev-site-example';
+import ResponsiveListandCellGridExample from '../example/ResponsiveListandCellGridExample?dev-site-example';
+import ResponsiveListandHtmlTableExample from '../example/ResponsiveListandHtmlTableExample?dev-site-example';
+
+# Creating Item Collection
+
+Start by making a `ResponsiveElement` that will switch between its two sub-components, a `Table` version of the content when above a certain size and a `List` version of the same content as the default display. In this `ResponsiveElement`, we set 3 attributes. The first is the defaultElement which is just the `List` version of the content. The second is the `responsiveTo` attribute, which determines what the component compares itself to in order to resizes itself, its parent or the window. Terra's `ItemCollection` surrounds its `ResponsiveElement` in a `<div>` tag, and sets `responsiveTo` to `parent`. Finally the breakpoint you want to switch at is defined by setting one of the following attributes: `tiny`, `small`, `medium`, `large`, or `huge` to the `Table` version of the content.
+
+The following is a simplified example of what the code should look like.
+
+```javascript
+  <ResponsiveElement
+    responsiveTo="parent"
+    defaultElement={listDisplay}
+    medium={tableDisplay}
+  />
+```
+
+With the base `ResponsiveElement` now set up, making the two displays versions of content is next.
+
+## Creating the List version
+
+Start with a Terra `List` component and define two of its attributes. `dividerStyle` is used to determine if and where you want the border-color to be applied each of the children and can have the values 'none', 'standard', 'bottom-only'. `role` is only used if the children you have in `List` are selectable, and if they are is defined as 'listbox'.
+
+```javascript
+  const listDisplay = (
+  <List dividerStyle='none' role='listbox'>
+    {children}
+  </List>);
+```
+
+Next we build the children that are going into this `<List>`. Each child should be wrapped in `terra-list`'s `<Item>` and as display in an `<ItemView>` from `terra-clinical-item-view`.
```

Comment:
We should note a the top, this guide it to achieve the deprecated terra-clinical-item-collection. An ItemView wouldn't have to be used here to flip between a table and list view

Reply:
Addressed here https://github.com/cerner/terra-ui/pull/310/commits/2f0433b8a6f6fb97139694a22a60e0ad1862bf40?file-filters%5B%5D=.jsx&file-filters%5B%5D=.mdx#diff-b887d62719eff551778a1fc191b08382c949a96aa056b14797bdcf836a34f062R33